nbbuild/build.xml

```diff
@@ -1592,8 +1592,11 @@ It is possible to use -Ddebug.port=3234 -Ddebug.pause=y to start the system in d
             includes="${allmodules},${harness-modules},nbbuild${build.source.config.add.ide},nbi,libs.antlr3.devel,libs.junit4"
             excludes="nbi/engine/lib,nbi/infra/build/jvm,nbi/infra/sandbox,nbi/infra/server,nbi/infra/utils"
     />
-    <pathconvert property="source.dirs" pathsep="," refid="source.dirset">
-      <regexpmapper from="${nb_all}/(.*)$" to="\1/**/*" handledirsep="yes"/>
+    <pathconvert property="nb_all_slash" dirsep="/">
+      <path location="${nb_all}"/>
+    </pathconvert>
+    <pathconvert property="source.dirs" dirsep="/" pathsep="," refid="source.dirset">
+      <regexpmapper from="${nb_all_slash}/(.*)$" to="\1/**/*" handledirsep="yes"/>
```

Comment:
Given that the `${nb_all}` value is (should be) apparently used as literal, wouldn't it be easier to just quote it in the regexp ? Also consider to use `${file.separator}` instead of `/`. E.g. `from="\Q${nb_slash}${file.separator}\E(.*)$"`

(warning: didn't try on Windows yet, just OTOH)

Reply:
I tried your regex and it will result in an empty `${source.dirs}`, so it is not working your way